views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
```

Comment:
Thanks for explaining what's happening with this style. Since I can't "see" that effect when I'm looking at the calculator in my browser, it was hard to see that it wasn't the best option. Also, I totally thought flexbox was a library, LOL.